### PR TITLE
feat(blog): CloudQuery with GCP and Kubernetes

### PR DESCRIPTION
**Pull request by SCKelemen**
Blog Post
Getting Started with GCP and Kubernetes

Checklist

- [x] Author profile
- [x] Author image
- [x] Blog post
- [x] Post images
- [x] Blog metadata
- [x] Checks pass

### CODE REVIEW

**Comment by erezrokah** [comment]
Looks like the Website build is failing due to a missing image:
https://vercel.com/cloudquery/cloudquery-web/J2n2tuk6bYf7ZkM22LxCWw6mrYEp

`Module not found: Can't resolve '../img/estore.jpg' in '/vercel/path0/website/pages/blog'`

**Comment by SCKelemen** [comment]
I ran the images through tinypng, but they're still >250kb.

File: website/pages/blog/getting-started-with-gcp-and-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+ - [psql]()
+
+
+
+
+Let’s start by creating a kubernetes cluster on GKE.
+Kubernetes clusters are a pool of compute resources, on which workloads can be scheduled.
+Clusters are a set of machines tied to a specific GCP region.
+
+To create a cluster, you'll need the project name, the GCP region, and a name for your cluster.
```
Comment (erezrokah) [inline website/pages/blog/getting-started-with-gcp-and-kubernetes.md:35]:
```suggestion
To create a cluster, you'll need the project name, the GCP region, add a name for your cluster.
```

Not sure if I got this right

File: website/pages/blog/getting-started-with-gcp-and-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+${project-name} = cq-kelemen
```
Comment (erezrokah) [inline website/pages/blog/getting-started-with-gcp-and-kubernetes.md:36]: Can we wrap this with a code block (```)? I think it's making Nextra (our build framework) angry

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Good stuff. I especially liked to `changeds-to-k8s-auth` as it's a bit more advanced.

I think we need to turn this blog just into `k8s` recipe under https://www.cloudquery.io/docs/recipes/overview 

I don't think how to run k8s is needed here due to if someone needs to use cq then they already familiar with k8s and if not then I think it's a whole another blog of intro to k8s which can be very long or even a series of blogs. so would keep it just as a recipe for gcp+k8s and this will also include the `changes-to-kubernetes-auth` potentially as a section/note

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**